blog.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+    <script src="./adopt.js" charset="utf-8"></script>
```

Comment:
I would suggest moving your script tags right before your </body>.  In this page, it would be line 81.

Reply:
Sorry, I don't think I understand what you mean by "right before your . ". Line 81 on the blog.html page is between the closing footer tag and the closing body tag. Is there where you mean for all the pages?

Reply:
It's sometimes good to have your script tag load after all of your HTML loads. By moving your script tag to just above your closing body tag. The browser will load all of your HTML before loading your JS. At this point in our learning, this is best case, just in case the loading of JS is slow. More advanced frameworks are a little different, but we're not there yet.

Reply:
Sounds like you recommend I move it then. I can make the update for Lesson 8 unless it's affecting Lesson 7. Please advise.

Reply:
No it's fine. It's not the end of the world. Go ahead and move it for Lesson08. It won't really matter for this class, but if you do some more advanced jQuery it might be useful to avoid page flash.